### ISSUE TITLE
Dracula theme support

### ISSUE THREAD

**Issue by JaeungHyun** [rationalmatter/Juno-Issue-Tracker#175]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

Hello, 
I think most developer prefer dracula theme for their IDEs.
I hope Juno supports dracula theme soon

Thanks!

**Comment by navoshta** [comment]
Thanks for reporting @JaeungHyun — yep I do plan to add more themes to Juno apps in future, will make sure to include a few options. 🙂

**Comment by MarcoLeder** [comment]
Hello - I also have something which would fit in:

Is it possible to have the same code marking (like the colors) as when opening Jupyter in the webbrowser? I have gotten quite used to the look and now with Juno Connect I am not able to tell commands and arguments appart as good as when using the Webbrowser. Could you please enable this option so it just takes over the colorsheme 1:1?

Best regards
Marco Leder

**Comment by navoshta** [comment]
@MarcoLeder Yep I agree that when theme support is added, a default Jupyter colour scheme should be an option.
